spec/shared_selenium_session.rb

```diff
@@ -302,6 +302,15 @@
       end
     end
 
+    describe '#send_keys' do
+      it 'defaults to sending keys to the document.activeElement' do
+        session.visit('/form')
+        session.send_keys(:tab)
```

Comment:
This should pre-verify the active element - otherwise the test isn't really proving anything